Subject: [PATCH] MountService: catch IllegalStateException for getField

An IllegalStateException will be thrown when getFiled is timeout.
So catch it to avoid crash.

diff --git a/services/core/java/com/android/server/MountService.java b/services/core/java/com/android/server/MountService.java
index 4cc2931..624520a 100644
--- a/services/core/java/com/android/server/MountService.java
+++ b/services/core/java/com/android/server/MountService.java
@@ -1007,6 +1007,8 @@
         String systemLocale;
         try {
             systemLocale = getField(StorageManager.SYSTEM_LOCALE_KEY);
+        } catch (IllegalStateException e) {
+            return;
         } catch (RemoteException e) {
             return;
         }
